samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js

```diff
@@ -178,7 +180,7 @@ const StorybookDecorator = (story, config = {}) => {
 	return (
 		<Theme
 			className={classnames(classes)}
-			title={`${config.kind}`.replace(/\//g, ' ').trim()}
+			title={componentName === config.name ? `Sandstone ${componentName}` : `${componentName} ${config.name}`}
```

Comment:
How about
```suggestion
			title={componentName === config.name ? `${config.kind}`.replace(/\//g, ' ').trim() : `${componentName} ${config.name}`}
```
for other theme usage?

Reply:
I think that's better than before. Thank you!
It's fixed.